### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by sledsworth**
I opened this mainly to update `esbuild` dependency to support `0.19.x`.

I updated all minor and patches, as well as `lit` while I was at it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,37 +28,37 @@
...
     "cross-env": "^7.0.3",
     "cz-conventional-changelog": "^3.1.0",
-    "esbuild": "^0.16.17",
+    "esbuild": "^0.19.5",
     "esbuild-plugin-alias": "^0.2.1",
     "esm": "^3.2.25",
-    "globby": "^13.1.4",
-    "lit": "^2.7.2",
+    "globby": "^13.2.2",
+    "lit": "^3.0.0",
```
Comment (bennypowers) [inline package.json:47]: This should maybe be a peerdep range so as to support lit 2 as well

**Comment by eatsjobs** [comment]
any chance to have this one merged?

**Review by bennypowers — CHANGES_REQUESTED** [review]
looks good, just please revert the whitespace changes, then we'll be ready to go

**Review by bennypowers — APPROVED** [review]
Thank you very much for the PR

**Pull request merged**